src/cjs.ts

```diff
@@ -51,31 +51,17 @@ export function interopDefault(
     return opts.preferNamespace ? sourceModule : defaultValue;
   }
   for (const key in sourceModule) {
-    if (key === "default") {
-      try {
-        if (!(key in defaultValue)) {
-          Object.defineProperty(defaultValue, key, {
-            enumerable: false,
-            configurable: false,
-            get() {
-              return defaultValue;
-            },
-          });
-        }
-      } catch {}
-    } else {
-      try {
-        if (!(key in defaultValue)) {
-          Object.defineProperty(defaultValue, key, {
-            enumerable: true,
-            configurable: true,
-            get() {
-              return sourceModule[key];
-            },
-          });
-        }
-      } catch {}
-    }
+    try {
+      if (!(key in defaultValue)) {
+        Object.defineProperty(defaultValue, key, {
+          enumerable: key !== "default",
+          configurable: key !== "default",
+          get() {
+            return sourceModule[key];
```

Comment:
There is a small chance of behavior regression here. 

In the current implementation, we were using the (cached) value of `sourceModule.default` via `defaultValue` now we dynamically access it.

Reply:
Unless the supplied module comes with interop code (which already defines `default` as a getter), I don't anticipate any changes in behavior here.

Furthermore, the appropriate behavior is either to cache all keys (getter) or not cache any at all, instead of caching `default` only.